docs/snippets/setup-slack-integration.mdx

```diff
@@ -121,6 +121,8 @@ Here is the format in the yml itself:
 slack:
   token: <your_slack_token>
   channel_name: <slack_channel_to_post_at>
+
```

Comment:
Can we have the `# optional #` section like we have below and have the value `<timezone_for_timestamps>` to keep consistent?

Reply:
Added the `# Optional #` section.